### ISSUE TITLE
Optional positions make mask incomplete when omitted

### ISSUE THREAD

**Issue by thomstark** [RobinHerbots/Inputmask#814]
```
$("div.height-field input").inputmask({mask:"9'9[9]\"", greedy:false, skipOptionalPartCharacter: "", "clearIncomplete":true});

$("div.weight-field input").inputmask({mask:"99[9]lbs", greedy:false, skipOptionalPartCharacter: "", "clearIncomplete":true});
```

If I omit the optional [9] when typing in either of these fields, it always returns incomplete on blur, and the field is cleared. I've also tried it this way:

```
$("div.height-field input").inputmask({mask:"9'9{1,2}\"", greedy:false, skipOptionalPartCharacter: "", "clearIncomplete":true});

$("div.weight-field input").inputmask({mask:"99{1,2}lbs", greedy:false, skipOptionalPartCharacter: "", "clearIncomplete":true});
```

Same results. It gets interpreted as incomplete when only one of the two possible numbers is entered, and thus the field clears on blur. I've had to remove `"clearIncomplete":true` from the config for the time being.

**Comment by RobinHerbots** [comment]
@thomstark , 

Fixed with the latest commit.

Depending what you include from extensions, you will need to escape the s in lbs as s is a definition in the date extensions.

A new release will follow soon.

**Comment by thomstark** [comment]
Thank you. I just installed the latest commit, and unfortunately the results are no different. It's still clearing the field if I enter say, 5'2", instead of 5'02". Same with 95lbs, instead of 095lbs. The ones one digit shorter are getting cleared on blur. Maybe I installed the wrong thing, but I did a diff checker on the js file and it's definitely the new commit.

**Comment by thomstark** [comment]
"A new release will follow soon." That probably means you haven't released the fix yet. :/ My bad.

**Comment by RobinHerbots** [comment]
Hmm, the latest commit should fix it. I tested your masks with qunit. Maybe
caching?  Can you make a jsfiddle with the masks.

Op 00:17 DI 3 Feb 2015 schreef thomstark notifications@github.com:

**Comment by thomstark** [comment]
I'll do that right now.

**Comment by thomstark** [comment]
Here's the fiddle: http://jsfiddle.net/6v4rr4rk/
Same behavior here, clears when missing the optional digit. In Chrome, IE, and Firefox. I'm loading the inputmask.js file as an external resource.

**Comment by thomstark** [comment]
Here's another one with the alternate method, same results: http://jsfiddle.net/6v4rr4rk/1/

**Comment by RobinHerbots** [comment]
@thomstark , 

Sorry, fixed in version 3.1.60

http://jsfiddle.net/6v4rr4rk/6/

**Comment by thomstark** [comment]
Works perfectly. Thanks a million for the fix, and the plugin.
